lib/mongoCrudHandler.js

```diff
@@ -87,13 +87,13 @@ module.exports = function init(config) {
     db.seagull.insert({_id: pair.id, value: crypted}, {w: 1}, function (err, result) {
       if (err) {
         if (err.code == 11000) {    // this is a mongo error for duplicate key
-          done({statuscode: 400, message: 'duplicate key', detail: err}, null);
+          done({statusCode: 400, message: 'duplicate key', detail: err}, null);
```

Comment:
True, nothing external of the API. However, while I was developing/testing the code, I got bit by `statuscode` versus the more common `statusCode`.  Since JavaScript isn't compiled/type-safe, then only way I exposed the problem was via thorough testing.  I figured I'd just take out one more potential problem while we support seagull.